src/parser/content-evaluator-module.ts

```diff
@@ -48,34 +49,73 @@ export class ContentEvaluatorModule implements Module {
 
   async _processComment(comments: Readonly<GithubCommentScore>[], specificationBody: string) {
     const commentsWithScore: GithubCommentScore[] = [...comments];
-    const commentsBody = commentsWithScore.map((comment) => comment.content);
-    const relevance = await this._evaluateComments(specificationBody, commentsBody);
 
-    if (relevance.length !== commentsWithScore.length) {
-      console.error("Relevance / Comment length mismatch! Skipping.");
-      return [];
+    const fixedRelevances: { [k: string]: number } = this._relevancesForCommentTypes();
```

Comment:
Wouldn't this be redundant in the configuration?

Reply:
I think you asked for it in the comment

> From my understanding, any comment within the pull request should have its relevance set to 1 because comments in the PR don't related to the specification itself, so doesn't make sense to evaluate them against the spec. Probably this should be configurable though the configuration file.


Some of these configured fixed relevances will go away when we implement https://github.com/ubiquibot/conversation-rewards/issues/45 

Overall, I think having the ability give fixed relevance to any comment type in the config, and getting it ignored by openai is a useful feature. But doesn't appear to be necessary at this stage.

Reply:
So this is still present. Just to let you know. If it is important to remove it from config, then it will need some more time.